multi-agent/tests/e2e/test_session_stress.py

```
@@ -265,6 +300,72 @@ def get_session_state(self, session_id: str) -> Dict:
+            raise RuntimeError(f"resource.save returned no rid: {result}")
+        return rid
+
+    def delete_resource(self, resource_id: str) -> bool:
+        """Delete a catalog resource by id."""
+        url = f"{self.base_url}/resources/resource.delete"
+        params = {"resourceId": resource_id}
+
+        try:
+            response = self.session.delete(url, params=params)
+            response.raise_for_status()
+            return True
```

Ruff (0.16.1)

[warning] Consider moving this statement to an `else` block

(TRY300)